webvi-systemlink-plugin/webvi-systemlink-plugin.lvproject

```diff
@@ -1,21 +1,22 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
conf files moved out of the WebApp.gcomp

![image](https://user-images.githubusercontent.com/1588923/87608166-c9d09080-c6c4-11ea-85f0-abb7e41cf41c.png)